typescript/src/pages/index.tsx

```diff
@@ -1,15 +1,61 @@
-import Link from 'umi/link';
-import Welcome from '@/components/Welcome';
-import styles from './index.css';
+import Link from "umi/link";
+import Welcome from "@/components/Welcome";
+import styles from "./index.css";
+import { connect } from "dva";
+import { GlobalState } from "@/common/type";
+import { actions } from "@/models/counter";
+import { bindActionCreators } from "redux";
+
```

Comment:
tsx 需要引入 React 来使用 `React.createElement()`，否则会报错 `'React' refers to a UMD global, but the current file is a module.`
```suggestion
import React from 'react';

```